TemplateApplication/Contributions/PackageCell.swift

```diff
@@ -55,20 +55,18 @@ struct PackageCell: View {
 
 
 #if DEBUG
-struct PackageCell_Previews: PreviewProvider {
-    static var previews: some View {
-        let mockPackage = Package(
-            name: "MockPackage",
-            version: "1.0",
-            branch: nil,
-            revision: "0",
-            // We use a force unwrap in the preview as we can not recover from an error here
-            // and the code will never end up in a production environment.
-            // swiftlint:disable:next force_unwrapping
-            repositoryURL: URL(string: "github.com")!,
-            license: "MIT License"
-        )
-        return PackageCell(package: mockPackage).previewLayout(.sizeThatFits)
-    }
+#Preview {
+    let mockPackage = Package(
+        name: "MockPackage",
+        version: "1.0",
+        branch: nil,
+        revision: "0",
+        // We use a force unwrap in the preview as we can not recover from an error here
+        // and the code will never end up in a production environment.
+        // swiftlint:disable:next force_unwrapping
+        repositoryURL: URL(string: "github.com")!,
+        license: "MIT License"
+    )
+    return PackageCell(package: mockPackage).previewLayout(.sizeThatFits)
```

Comment:
Small styling, but I would suggest to add a line break here:
```suggestion
    )
    
    return PackageCell(package: mockPackage)
```